src/services/middlewareServices.js

```diff
@@ -1,16 +1,13 @@
-import { list, create, watch, update, currentUser, OpenShiftWatchEvents } from './openshiftServices';
+import { watch, update, currentUser, OpenShiftWatchEvents } from './openshiftServices';
 import { middlewareTypes } from '../redux/constants';
 import { FULFILLED_ACTION } from '../redux/helpers';
 import { buildServiceInstanceResourceObj, DEFAULT_SERVICES } from '../common/serviceInstanceHelpers';
 import {
   buildValidProjectNamespaceName,
-  namespaceRequestDef,
-  namespaceDef,
-  namespaceRequestResource,
-  statefulSetDef,
-  routeDef,
-  secretDef
+  findOpenshiftResource,
+  findOrCreateOpenshiftResource
 } from '../common/openshiftHelpers';
+import * as osKinds from '../common/openshiftKinds';
```

Comment:
Can we just import what is needed instead of all the exports?  What do you think?